homeassistant/components/osoenergy/services.yaml

```diff
@@ -0,0 +1,257 @@
+turn_on:
```

Comment:
Please split the services out into a new PR.

I didn't know that services should be added to its own PR to keep the "add integration" PR as small as possible. Sorry still new to the review process

Reply:
I have removed the services for this PR and will add them as you have written in another PR after this one is approved and merged. I will do the same for the documentation.